CMakeLists.txt

```diff
@@ -128,7 +128,7 @@ set(PKG_CONFIG_DESCRIPTION_SUMMARY "C++ framework for property based testing ins
 set(PKG_CONFIG_VERSION)
 set(PKG_CONFIG_LIBDIR "\${prefix}/lib")
 set(PKG_CONFIG_INCLUDEDIR "\${prefix}/include")
-set(PKG_CONFIG_LIBS)
+set(PKG_CONFIG_LIBS "-lrapidcheck")
```

Comment:
```suggestion
set(PKG_CONFIG_LIBS "-L${libdir} -lrapidcheck")
```

Reply:
Per https://github.com/emil-e/rapidcheck/pull/319/files#r1523528740

Reply:
It did not appear necessary, and isn't this redundant with `set(PKG_CONFIG_LIBDIR "\${prefix}/lib")` above?
Also should that line be `set(PKG_CONFIG_LIBDIR "\${libdir}")` then?

Reply:
> It did not appear necessary,

I think that is because Nixpkgs adds all the `-L` flags. (BTW I think we should split out that part of the CC wrapper hook, so packages can opt to relying on proper pkg-configs a bit harder.)

> and isn't this redundant with `set(PKG_CONFIG_LIBDIR "\${prefix}/lib")` above?

No because I think that is for *defining* `libdir=....`

> Also should that line be `set(PKG_CONFIG_LIBDIR "\${libdir}")` then?

Then we could get the impredicative
```pkg-config
libdir=${libdir}
```

What we want to end up with is 

```pkg-config
Libs: -L${libdir} -lrapidcheck
```

Reply:
> Nixpkgs adds all the `-L` flags.

Makes sense, so I've applied your suggestion.

> No because I think that is for _defining_ `libdir=....`

Ah, of course. I didn't see through the indirection and I thought this referred to some implicit CMake variables. It doesn't, and that's pretty obvious from the `.pc` contents.

Thanks!